ParisBNF/abb/BNFabb107.xml

```diff
@@ -29,10 +29,214 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                            </incipit>
+                            <explicit xml:lang="gez">ወርእየ፡ እግዚአብሔር፡ ሕማሞሙ፡ ሐውፃሙ፡ እግዚኦ፡ ለደቂቀ፡ እስራኤል፡ አድኅኖሙ፡ በይእቲ፡ ዕለት፡ ከማሁ፡ አድኅነኒ፡ አምላኪየ፡ አምላከ፡ አበውየ፡ ሰምዓኒ፡ እግዚኦ፡ ጸሎትየ፡ እምኵሉ፡ ምንዳቤየ፡ ወእምኵሉ፡ ትካዝየ፡ ለገብርከ፡ ባውላ።</explicit>
+                            <listBibl type="text">
+                                <bibl>
+                                    <ptr target="bm:Aescoly1951Recueil"/>
+                                    <citedRange unit="page">170-175</citedRange>
+                                </bibl>
+                            </listBibl>
+                        </msItem>
+                        <msItem xml:id="ms_i14">
+                            <locus from="50va" to="75vb"/>
+                            <title corresp="LIT7504PrayerBE"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ </hi>በእንተ፡ ስብሐት፡ ወበእንተ፡ ሕፃናት፡ አእምሩ፡ <hi rend="rubric">ስብሐትየ። በዝንቱ፡ መ</hi>ጽሐፍ፡ እግዚእየ፡ እግዚእየ፡ በእንተ፡ ስምከ፡ ቅዱስ፡ መሐረኒ፡</incipit>
+                            <explicit xml:lang="gez">ዘኵሉ፡ ዘይጼውዕ፡ ስመ፡ እግዚአብሔር፡ ይድኅን፡ በጽድቅ፡ ለገብርከ፡ <hi rend="rubric">ስምዖን</hi>። ተፈጸመ፡ መጽሐፈ፡ ፈላሻ፡ በዘመነ፡ ሉቃስ፡ በወርኅ፡ ጥምት፡ በትዕይንተ፡ ጎንደር፡ ዓፄ፡ ሣህሉ፡ ታእካ፡ እንዘ፡ የዓቅቡ፡ በጥቅምት፡ በ፫ዕለት።</explicit>
+                           <listBibl type="text">
+                                <bibl>
+                                    <ptr target="bm:Aescoly1951Recueil"/>
+                                    <citedRange unit="page">198-271</citedRange>
+                                </bibl>
+                           </listBibl>
+                        </msItem>
+                        <!--  <msItem xml:id="ms_i15">
+                            <locus from="75vb" to="119va"/> 
+                            <title ref="LIT2436Teezaz"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">ይትባረክ፡ እግዚአብሔር፡ አምላክ፡ ለኵሉ፡ መንፈስ፡ ወለኵሉ፡ ዘሥጋ፡ ዝውእቱ፡ አዕፅምቲሆሙ፡ ለደቂቀ፡ እስራኤል፡ እለ፡ ቦሙ፡ ብሔረ፡ ግብጽ፡ ምስለ፡ ያዕቆብ፡ አቡሆሙ፡</hi></incipit>
+                        <explicit xml:lang="gez">ወአምከኖ፡ እግዚአብሔር፡ ለአብርሃም፡ በእንተ፡ ይስሐቅ፡ ወልዱ፡ ወይቤ፡ ቅትል፡ ወልድከ፡ ዘታፈቅር፡ ይስሐቅሃ፡ ወሡዕ።
+                        <lb/>
+                            <hi rend="rubric">ቀደማዊ፡ ቤገበ፡ ውሉደ፡</hi> ባሮክ፡ ውሉደ፡ አባ፡ ኢርማዲርማ። ወልደ፡ አባ፡ ስማርቸጊን። ውሉዱ፡ አባ፡ አማን። ውሉደ፡ አባ፡ ያዕቆብ። ውሉደ፡ አባ፡ በለዊ። ወለዘአንበቦ፡ ወለዘተርጐሞ፡ ወለዘሰምዓ፡ ቃላቲሁ፡ ኅቡረ፡ ይምሐረነ፡ <hi rend="rubric">እግዚአብሔር።</hi></explicit>
+                           <note>The edited text by Halevy on pp. 1 corresponds to what is in the ms on f. 105r</note>
+                        </msItem>-->
+                    </msContents>
                     <history>
                             <origin>
                                 <origDate notBefore="1837" notAfter="1848" source="lettering">19th century</origDate>.  
```

Comment:
If the date can be known so precisely, I would include the same range in the content (or close the element)